src/util/c_types.h

```diff
@@ -36,6 +36,27 @@ class c_bit_field_typet : public bitvector_typet
   {
     return subtype();
   }
+
+  // Use .underlying_type() instead -- this method will be removed
+  const typet &subtype() const
+  {
+    // The existence of get_sub() front is enforced by check(...)
+    return static_cast<const typet &>(get_sub().front());
+  }
+
+  // Use .underlying_type() instead -- this method will be removed
+  typet &subtype()
+  {
+    // The existence of get_sub() front is enforced by check(...)
+    return static_cast<typet &>(get_sub().front());
+  }
+
+  static void check(
+    const typet &type,
+    const validation_modet vm = validation_modet::INVARIANT)
+  {
+    type_with_subtypet::check(type, vm);
```

Comment:
This function should include a call to `bitvector_typet::check` as well, because the new function hides the existing one.